app/internal/security/dependancies.py

```diff
@@ -1,8 +1,10 @@
 from starlette.requests import Request
 
+from app.database.models import User
 from app.dependencies import get_db
 from app.internal.security.ouath2 import (
-    Depends, Session, check_jwt_token, get_authorization_cookie)
+    Depends, Session, check_jwt_token, get_authorization_cookie,
+    HTTPException, HTTP_401_UNAUTHORIZED)
```

Comment:
Close in a separate line, order the imports

Reply:
Why not import HTTP directly?